client/src/components/App.jsx

```diff
@@ -25,37 +25,33 @@ class App extends React.Component {
   }
 
   handleGetClickInfo (e) {
-    var arrayOfWidgets = ['review-container', 'overview', 'RICWid', 'header'];
     var time = moment().format().toString();
     var widgetName;
-
+    var elementType = e.target.nodeName;
     checkClasses(e.target);
+    Atelier.logClick(elementType, widgetName, time);
 
     function checkClasses (element) {
       var currentClassNames = element.className;
       if (currentClassNames === 'overview') {
-        console.log("Im the overview!");
         widgetName = 'OVERVIEW';
```

Comment:
You can combine the return statement with the line. return widgetName = 'Overview';